sdk/src/main/scala/com/horizen/SidechainApp.scala

```diff
@@ -155,8 +177,13 @@ class SidechainApp @Inject()
   if(sidechainSecretStorage.isEmpty) {
     for(secretHex <- sidechainSettings.wallet.genesisSecrets)
       sidechainSecretStorage.add(PrivateKey25519Serializer.getSerializer.parseBytes(BytesUtils.fromHexString(secretHex)))
+
+    for(secretSchnorr <- sidechainSettings.backwardTransfer.backwardTransferSecrets)
+      sidechainSecretStorage.add(SchnorrSecretKeySerializer.getSerializer.parseBytes(BytesUtils.fromHexString(secretSchnorr)))
```

Comment:
according to the comment above
```
for(secretHex <- sidechainSettings.wallet.genesisSecrets)
      sidechainSecretStorage.add(PrivateKey25519Serializer.getSerializer.parseBytes(BytesUtils.fromHexString(secretHex)))

for(secretSchnorr <- sidechainSettings.backwardTransfer.backwardTransferSecrets)
      sidechainSecretStorage.add(SchnorrSecretKeySerializer.getSerializer.parseBytes(BytesUtils.fromHexString(secretSchnorr)))
```

can be replaced with:
```
for(secretHex <- sidechainSettings.wallet.genesisSecrets)
      sidechainSecretStorage.add(sidechainSecretsCompanion.parseBytes(BytesUtils.fromHexString(secretHex)))
```

Reply:
prefer to do it as is